backend/api.py

```diff
@@ -49,42 +43,35 @@ class HintResponse(BaseModel):
     allow_headers=["*"]
 )
 
-# Create the main endpoint
 @app.get("/")
 async def root() -> str:
-    print("Welcome to LeetCopilot!")
     return "Welcome to LeetCopilot!"
 
-@app.get('/fact')
-async def health() -> str:
-    global fact_index
-    fact_index += 1
-    return ['LeetCopilot will be a huge hit! Guaranteed🪓', 
-            'LeetCopilots founder, Ridha, is very cool😎', 
-            'LeetCopilots developers are certified stinky👃'][fact_index % 3]
-
 # LLM Dependency injection
 def get_llm() -> OpenAI:
-    load_dotenv()
     openai_api_key = os.getenv("OPENAI_API_KEY")
     return OpenAI(openai_api_key=openai_api_key, temperature=0.5, max_tokens=1024)
 
-# Endpoint to get a leetcode hint
-@app.post("/hint", response_model=HintResponse)
-async def get_hint(problem: HintRequest, llm: OpenAI = Depends(get_llm)):
+@app.post("/hint", response_model=str)
```

Comment:
I generally have a bias towards returning objects in APIs and avoiding primitives. Returning objects sets us up better in the future to add new fields etc. Open to keeping it like this though for the time being

Reply:
Yeah for sure I get what you're saying. I was just thinking we can switch it when needed. For now it can be left like this and once we have a proper object to return, likely once we are ready to return line by line tips, we can switch to objects.